src/Security/OpenIdConfigurationProvider.php

```diff
@@ -192,10 +197,32 @@ public function getAuthorizationUrl(array $options = []): string
      *
      * @throws ItkOpenIdConnectException
      */
-    public function validateIdToken(string $idToken, string $nonce): object
+
+    /**
+     * Do any required verification of the id token and return an array of decoded claims
+     *
+     * @param string $idToken
+     *   Raw id token as string
```

Comment:
Let's drop "as string", it's typehinted, so really don't know why I wrote it.

Reply:
Fixed in 860c85a